rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction.adoc

```diff
@@ -0,0 +1,107 @@
+
+## Example Scenario
+
+Imagine you work as an accountant at Mythical University. Mythical University has ordered a travel package for one of their field excursions. You have received an email and saved the attached order confirmation to your computer.
+
+This is what an order confirmation of your travel agency Wonderlust Travel typically looks like:
+
+[[confirmationletter-image]]
+image::usecase-1-wonderlusttravels-orderconfirmation-pdf.png["Letter containing a logo, headers, text and tables. The order ID is contained in a table in a header in the middle of the page. The label says 'Order ID'.", 75%]
+
+You need to search the order ID in your order tracking system to get the corresponding tracking number, which you need to reference in your payment.
+
+Unfortunately your order tracking system is very old and accountants can only access it by typing in and reading from a terminal screen:
+
+[[trackingsystem-allentries-image]]
+image::as400ordertrackingscreen.png["Legacy order tracking system showing green glowing text in a terminal. Above a list of orders, there is an entry field for the order id labeled 'Order-ID' and a button labeled 'Search' next to it.", 75%]
+
+You click the search field, enter the order ID, and click 'Search'.
+
+[[trackingsystem-noentries-image]]
+image::as400ordertrackingscreen-searchresult-ordernotfound.png["Legacy order tracking system shows no search results.", 75%]
+
+The list of orders is empty now. This could either mean that the order is'nt there or that you mistyped the number. Let's assume that the latter is the case. You try again and the list shows the corresponding entry:
+
+[[trackingsystem-resultentry-image]]
+image::as400ordertrackingscreen-searchresult-orderfound.png["Legacy order tracking system shows one search result.", 75%]
+
+Now you can copy the tracking number and reference it in your payment.
+
+Since all the scientists at Mythical University travel a lot, you have to repeat this tedious, error-prone and risky process several times per day. 
+
+You decide to get a bot to do this.   
+
+The following example shows how you automate the retrieving of the order ID from a PDF and of the tracking number from the legacy system using MuleSoft RPA as an invocable automation step. How to download the right attachment before and how to proceed afterwards isn't part of this example.
+
+## Design a Model of the Process in RPA Manager
+
+In RPA Manager, you open a new project and design a model of the process.
+
+The model contains the following activities:
+
+. Extract the order ID from the order confirmation.
+. Extract the corresponding tracking number from the legacy system.
+. Return either the tracking number or an information that it couldn't be found.
+
+image::extractrackingnumber-bpmn.png["In the flow, activities one and two are sequential. Depending on whether the order id is found, an exclusive gateway branches the flow to the third activity."]
```

Comment:
I think the important part of the image is the activities, not the description of the decision tree.

A flow showing extracting the order ID and then the tracking ID from the order confirmation...

Reply:
Does this graphic follow Salesforce guidelines?

Reply:
The modelling of the decision tree is as important as the modelling of the activities. This is a screenshot of our product showing the model in BPMN standard (Business Process Modelling and Notation).

Reply:
I added that information together with a link to the paragraph.